concordium-cis2/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ## [6.2.1-alpha.1] - 2025-10-03
 
-- Added release-concordium-cis2.yaml to trigger github release action
+- RUST VERSION bumped up to 1.8.5
```

Comment:
To stay consistent with the rest of the `changelog`. The version is `1.85`

- Bump MSRV (minimal-supported rust version) to 1.85